### DISCUSSION TITLE
https://github.com/phrizm/bs-ukijo-parallax

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [bootscore/bootscore#355] [Show and tell]
Hello Basti, take a look here,
https://github.com/phrizm/bs-ukijo-parallax

Thanks

**Comment by crftwrk** [comment]
Yes, but what it does and how it works? 😉

**Comment by ritchiemarkov** [comment]
Sorry my bad. Drop into your child ofcourse.. Enqueveu in functions php
file. Then add an image and add custom css class to it. "parallax
width-100".

It's fires up ukiyo.js with pretty smooth parallax effect.

https://www.northwestalps.com/2023/01/11/parralax-tricked/


Il Ven 13 Gen 2023, 17:56 Basti ***@***.***> ha scritto:
